### DISCUSSION TITLE
TPMS how to add extra set of wheels/ sensors ?

### DISCUSSION THREAD

**Discussion by DavidIgor** [cedricp/ddt4all#994] [General]
![image](https://github.com/cedricp/ddt4all/assets/122531259/3b4ee94d-d849-4c21-b69c-9e504afb3c72) I got extra set of wheels for winter and I want to have all 8.  why in ddt4all  I can see 4 sensor ID twice? can I just change bottom 4 to new ones?

**Comment by matzel82** [comment]
What a Car you're own yours ??

**Reply by DavidIgor** [reply]
dacia dokker 2018 H4M

**Reply by matzel82** [reply]
It's only available 1 set of tire. You can't save 8 tires. If you found a way...tell me ❤️

**Reply by DavidIgor** [reply]
so I have to change sensor IDs with new set of wheels? at winter and summer time...

**Reply by KarelSvo** [reply]
It's easier than you think.
The tire pressure reset function in the on-board computer saves the air pressure as well as the new sensor ID.

**Reply by DavidIgor** [reply]
> It's easier than you think.
> 
> The tire pressure reset function in the on-board computer saves the air pressure as well as the new sensor ID.

no need for ddt4all? just put new wheels on and reset the tpms? even if new wheels has brand new sensors?

**Reply by KarelSvo** [reply]
Yes, if the sensor type is compatible with the vehicle.  All Dacia with T4 electronics and TPMS sensors work like this.

**Reply by DavidIgor** [reply]
amazing news! thank you so much! I will try it. 
I guess Ill stick with turning on clock permanently on display with ddt4all . do you have any manual for this? I found one but mh car has different "TB Cluster " option then on mentioned manual.

**Reply by KarelSvo** [reply]
The easy way is, to use [this](https://transfiles.ru/855g5) software.
Connect interface, start software, select interface, click macro button and open x67 folder.
By clicking on the wish list, your wish will be sent to the control unit.

**Reply by DavidIgor** [reply]
I guess learning time for me... 
thanks for link.

**Reply by DavidIgor** [reply]
so my car Dacia Dokker 2018 (5seater) is x67?

**Reply by KarelSvo** [reply]
All Dokkers are x67 series. You can also see it when you click on scan. The most popular changes are summarized in the folder.

**Reply by DavidIgor** [reply]
copy that

**Reply by DavidIgor** [reply]
![Uploading 295195637...]I did it ad it doesn't seem to work... no clock on display ![image](https://github.com/cedricp/ddt4all/assets/122531259/2007c6db-fe24-4ed2-9f97-80244a351052)

**Reply by KarelSvo** [reply]
Have you checked all pages of the on-board computer?

**Reply by DavidIgor** [reply]
yes

**Reply by KarelSvo** [reply]
Your photo shows that all commands were executed correctly.
For versions with stop and start functions, the time cannot be displayed on the main page.
There are additional symbols at the bottom.
https://a.d-cd.net/32e51bes-960.jpg

**Reply by DavidIgor** [reply]
that is what I have ...  ah well... not even outside temperature ?  note: disabling complimentary rear wiper was disabled successfully!

**Reply by DavidIgor** [reply]
can i make my fog lights work as cornering lights?

**Reply by AndyIreland** [reply]
you might be able to - have you got 'cornering' showing up in your Edison EMM lighting ECU

**Reply by DavidIgor** [reply]
when i scan ECUs it shows only this: ![Uploading 295382221...] LIGHTING_by_emm_cf
